integration_tests/components/bedspaceConflictErrorComponent.ts

```diff
@@ -30,20 +30,22 @@ export default class BedspaceConflictErrorComponent {
     if (conflictingEntityType === 'booking') {
       Page.verifyOnPage(BookingShowPage, [this.premisesId, conflictingEntity as Booking])
     } else {
-      Page.verifyOnPage(LostBedShowPage, conflictingEntity as LostBed)
+      Page.verifyOnPage(OutOfServiceBedShowPage, this.premisesId, conflictingEntity as Cas1OutOfServiceBed)
     }
 
     cy.go('back')
   }
 
-  private getTitle(fields: Array<string>): string {
-    return fields.length === 1
-      ? 'This bedspace is not available for the date entered'
-      : 'This bedspace is not available for the dates entered'
+  private getTitle(fields: Array<string>, conflictingEntityType: 'booking' | 'lost-bed'): string {
+    return (
+      conflictingEntityType === 'lost-bed'
+        ? 'Out of service bed record cannot be created for the $date$ entered'
+        : 'This bedspace is not available for the $date$ entered'
+    ).replace('$date$', fields.length === 1 ? 'date' : 'dates')
   }
 
   private getMessage(fields: Array<string>, conflictingEntityType: 'booking' | 'lost-bed'): string {
```

Comment:
The type definition for `ConflictingEntityType` already exists elsewhere, doesn't it? It shouldn't have to be defined here